formserver/src/main/java/com/diyasylum/formserver/serializers/EncryptedMessageSerializer.java

```diff
@@ -0,0 +1,54 @@
+package com.diyasylum.encryption.serializers;
```

Comment:
This package name is wrong.

Reply:
Oops thanks. This is what I get for not using an IDE.